staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -354,6 +354,12 @@ type KubeletConfiguration struct {
 	// Default: "None"
 	// +optional
 	CPUManagerPolicy string `json:"cpuManagerPolicy,omitempty"`
+	// singleProcessOOMKill, if set, will prevent the `memory.oom.group` flag from being set for container
```

Comment:
```suggestion
	// singleProcessOOMKill, if true, will prevent the `memory.oom.group` flag from being set for container
```

something like that.